src/components/sbb-skiplink-list/sbb-skiplink-list.tsx

```diff
@@ -1,51 +1,50 @@
-import { Component, ComponentInterface, Element, h, JSX, State, Prop } from '@stencil/core';
-import { InterfaceTitleAttributes } from '../sbb-title/sbb-title.custom';
+import { TitleLevel } from '../sbb-title';
 import {
   createNamedSlotState,
   HandlerRepository,
   namedSlotChangeHandlerAspect,
 } from '../../global/eventing';
+import { CSSResult, html, LitElement, nothing, TemplateResult } from 'lit';
+import { customElement, property, state } from 'lit/decorators.js';
+import { spread } from '@open-wc/lit-helpers';
+import { SbbLink } from '../sbb-link';
+import style from './sbb-skiplink-list.scss?lit&inline';
+import '../sbb-title';
 
 /**
  * @slot - Use the unnamed slot to add `sbb-link` elements to this skiplink list.
  */
-@Component({
-  shadow: true,
-  styleUrl: 'sbb-skiplink-list.scss',
-  tag: 'sbb-skiplink-list',
-})
-export class SbbSkiplinkList implements ComponentInterface {
+@customElement('sbb-skiplink-list')
+export class SbbSkiplinkList extends LitElement {
+  public static override styles: CSSResult = style;
+
   /** The title text we want to place before the list. */
-  @Prop() public titleContent?: string;
+  @property({ attribute: 'title-content' }) public titleContent?: string;
 
   /** The semantic level of the title, e.g. 2 = h2. */
-  @Prop() public titleLevel?: InterfaceTitleAttributes['level'] = '2';
+  @property({ attribute: 'title-level' }) public titleLevel?: TitleLevel = '2';
 
   /** sbb-link elements */
-  @State() private _links: HTMLSbbLinkElement[];
+  @state() private _links: SbbLink[];
 
   /** State of listed named slots, by indicating whether any element for a named slot is defined. */
-  @State() private _namedSlots = createNamedSlotState('title');
-
-  @Element() private _element!: HTMLElement;
+  @state() private _namedSlots = createNamedSlotState('title');
 
-  private _syncLinks(): void {
-    this._element.querySelectorAll('sbb-link').forEach((link: HTMLSbbLinkElement) => {
+  public syncLinks(): void {
```

Comment:
This method probably should be private. or is there any reason to have it public?